docs/pratices/latency_slo_distrib_metric_cloud_monitoring.md

```diff
@@ -0,0 +1,107 @@
+
+Adapt the window size until you get data points, just one is enough.
+
+The response contains the latency distribution [Bucket Options](https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TypedValue#bucketoptions). (This information is not returned by the GET on the metric descriptor, c for that we need an inspect time series)
+
+This looks like:
+
+```json
+'bucketOptions': {'exponentialBuckets': {'growthFactor': 1.4142135623730951,
+                                         'numFiniteBuckets': 64,
+                                         'scale': 1.0}},
+```
+
+Once we have the type of buckets (Explicit, Linear, Exponential) we can calculate the min and max values ​​of each bucket of the distribution type metric.
+
+As an example let’s use AppEngine latency (see above): exponential type, grow factor is square root of 2, 64 finite buckets, so 66 in total (+ and - l infinity)
+
+ This [gSuite Sheet](https://docs.google.com/spreadsheets/d/1pvGC_BW1l0D1D8GJY8I3H4QL76xVQ8t0QF_dIQ5lg5I/edit?usp=sharing)  allows to calculate the border values ​​of each bucket of an exponential distribution knowing these parameters (growfactor, numFiniteBuckets and scale)
+
+**The SLI-SLO threshold MUST be aligned with one of the border values, since we are blind between 2 borders** (see the trade off distributive explained above).
+
+ In SLO Generator we directly indicate the number of the bucket corresponding to the value of the top fontiere of the bucket as in this [yaml example](https://github.com/GoogleCloudPlatform/professional-services/blob/master/tools/slo-generator/samples/stackdriver/slo_gae_app_latency.yaml)
+
+## D - Choose a border value for each latency SLO
+
+Console / Monitoring / metric explorer
+
+Select the service and the metric, with our example it is GAE / Response latency
+
+Take a Stacker Bar performance
+
+Duration of the graph = 6 weeks
+
+Aggregator 50th:
+
+![aggregator 50th](../images/latency_aggregator50th.png)
+
+The value of 6 ms returned by the graph and close in the [gSuite Sheet](https://docs.google.com/spreadsheets/d/1pvGC_BW1l0D1D8GJY8I3H4QL76xVQ8t0QF_dIQ5lg5I/edit?usp=sharing) to the 8 ms high border of bucket number 6
+
+Aggregator 99th:
+
+![aggregator 99th](../images/latency_aggregator99th.png)
+
+The value of 125 ms returned by the graph and close in the [gSuite Sheet](https://docs.google.com/spreadsheets/d/1pvGC_BW1l0D1D8GJY8I3H4QL76xVQ8t0QF_dIQ5lg5I/edit?usp=sharing) to the 128ms high border of bucket number 14
+
+We can define the following 2 latency SLOs by specifying the threshold_bucket value in the yaml config
+
+* **Latency 8 ms** (threshold_bucket **6**) SLO=50%
+* **Latency 128 ms** (threshold_bucket **14**) SLO=99%
+
+This will make 2 SLI-SLO definitions using two [yaml files](https://github.com/GoogleCloudPlatform/professional-services/blob/master/tools/slo-generator/samples/stackdriver/slo_gae_app_latency.yaml) in SLO Generator.
```

Comment:
```suggestion
This will make 2 SLI-SLO definitions using two [yaml files](https://github.com/google/slo-generator/blob/master/tools/slo-generator/samples/stackdriver/slo_gae_app_latency.yaml) in SLO Generator.
```